sdk/protocol-plugins/src/plugins/common/builders/SwapActionBuilder.ts

```diff
@@ -11,7 +11,7 @@ export const SwapActionBuilder: ActionBuilder<steps.SwapStep> = async (params):
 
   const swapData = await swapManager.getSwapDataExactInput({
     chainInfo: params.user.chainInfo,
-    fromAmount: step.inputs.fromTokenAmount,
+    fromAmount: step.inputs.swappedAmount,
```

Comment:
I'm not sure I like the term swapped amount.

It reads to me as the amount that's been swapped. So could mean the after swap amount. Tiny bit ambiguous for me but is a nit.

Reply:
maybe `amountAfterFee` or `amountToSwap` I don't know how to convey that this amount is amount after fee